pkg/services/authz/zanzana/common/tuple.go

```diff
@@ -245,3 +248,18 @@ func ToOpenFGATuples(tuples []*authzextv1.Tuple) []*openfgav1.Tuple {
 	}
 	return result
 }
+
+func AddRenderContext(req *openfgav1.CheckRequest) {
+	if req.ContextualTuples == nil {
+		req.ContextualTuples = &openfgav1.ContextualTupleKeys{}
+	}
+	if req.ContextualTuples.TupleKeys == nil {
+		req.ContextualTuples.TupleKeys = make([]*openfgav1.TupleKey, 0)
+	}
+
+	req.ContextualTuples.TupleKeys = append(req.ContextualTuples.TupleKeys, &openfgav1.TupleKey{
+		User:     RenderUser,
+		Relation: "view",
+		Object:   NewNamespaceResourceIdent("dashboard.grafana.app", "dashboards"),
```

Comment:
Nit: use constant exported but dashboards apis package like we do for folders https://github.com/grafana/grafana/blob/main/pkg/services/authz/zanzana/common/info.go#L16-L17